bot/exts/utils/package.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import random
+from typing import Dict, Union
+
+from aiohttp.client_exceptions import ContentTypeError
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+log = logging.getLogger(__name__)
+
+pypi_api_url = "https://pypi.org/pypi/{package_name}/json"
+
+
+class Package(commands.Cog):
+    """Provides Information about package hosted at pypi."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @staticmethod
+    def embed_builder(pypi_pkg: dict) -> Embed:
+        """Returns informative embed about package."""
+        embed = Embed()
+        embed.colour = Colours.soft_orange
+        embed.set_thumbnail(url="https://miro.medium.com/max/660/1*2FrV8q6rPdz6w2ShV6y7bw.png")
```

Comment:
We had a chat about this internally, and we're not comfortable with linking an asset hosted on medium.com partly because it's medium, and partly because of potential licensing issues. I imagine you have chosen to not link the [PyPI asset](https://pypi.org/static/images/logo-small.6eef541e.svg) directly because it's an SVG? I'm not sure what the best solution would be, but currently we'd rather have no logo than linking to medium.com.

Reply:
Can I use this [link ](https://raw.githubusercontent.com/github/explore/666de02829613e0244e9441b114edb85781e972c/topics/pip/pip.png)

Reply:
We decided to add the logo as an emoji on PyDis, feel free to use the CDN link: https://cdn.discordapp.com/emojis/766274397257334814.png